add support for osm and osm-edge. (#279)
merbridge/merbridge: deploy/README.md

helm install -n kuma-system --set mode=kuma merbridge helm
```

> ### Install Merbridge on Osm/Osm-edge
> 
> + set mode=osm to switch to osm/osm-edge mode
> + set namespace where merbridge is going to install by -n .

Reviewer: ```suggestion
### Install Merbridge on OSM/OSM-Edge

+ Set `mode=osm` to switch to [OSM](https://github.com/openservicemesh/osm)/[OSM-Edge](https://github.com/flomesh-io/osm-edge) mode
+ Specify a namespace where you want to install Merbridge using `-n`
```

Author: correct.
